package.json

```diff
@@ -95,7 +95,7 @@
     "codecov": "CODECOV=1 ./scripts/test/js_test.sh",
     "watch": "WATCH=1 ./scripts/test/js_test.sh",
     "flow": "flow check",
-    "fmt": "prettier-eslint --write --no-semi --ignore 'static/js/flow/**/*.js' 'static/js/**/*.js'",
+    "fmt": "LOG_LEVEL= prettier-eslint --write --no-semi --ignore 'static/js/flow/**/*.js' 'static/js/**/*.js'",
```

Comment:
cool, seems fine to me, I usually run the command locally so I hadn't seen the issue.